main.go

```diff
@@ -81,7 +82,9 @@ func init() {
 	pflag.String("http-proto", "https", "Set to 'http' if necessary")
 	pflag.String("http-user", "", "HTTP username for pulling the remote file")
 	pflag.String("http-pass", "", "HTTP password for pulling the remote file")
+
 	pflag.String("http-url", "", "Remote endpoint to retrieve the file from")
+	pflag.String("s3-obj", "", "Remote endpoint in S3 to retrieve the file from if s3-arn isn't used")
```

Comment:
still talks about an ARN.... either here or the docs should give examples of what this is. I'm sure what to put here

Reply:
Done.